fontique/src/backend/fontconfig/mod.rs

```diff
@@ -314,6 +301,17 @@ struct Config {
     lang_maps: Vec<(String, StyleClass, String)>,
 }
 
+impl Config {
+    pub fn cache_dirs(&self) -> Vec<PathBuf> {
+        if self.cache_dirs.is_empty() {
+            let xdg_dirs = xdg::BaseDirectories::with_prefix("fontconfig").unwrap();
```

Comment:
This does an awful lot of allocation, given that we only use one item from it.

Reply:
Removed